app/konnect/api-products/productize-service.md

```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
 {% navtabs %}
 {% navtab Konnect UI %}
 
-API products can be published to the Dev Portal. To publish your new API product navigate to the {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and follow these steps: 
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to. 
+    Product versions are published by default when you select Dev Portals, you can disable publishing while configuring the Dev Portals if you'd like to publish them later.
 
-1. Select the API product that you created in the previous step.
-2. Navigate to the **Actions** menu, then select **publish**. 
-
-Your API product is now consumable by developers from the **Dev Portal**.
+The API product and product versions should now display in the Dev Portals you selected.
 {% endnavtab %}
 {% navtab API %}
-You can publish an API product by issuing a `POST` request to the [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint. Ensure you replace `{ApiProductId}` with the API product ID returned in the previous step.. 
+
+1. Before you publish the API product version, you must first assign the API product to any Dev Portals by issuing a `PATCH` request to the [`/api-products/{id}](/konnect/api/api-products/latest/#/API%20Products/update-api-product) endpoint:
+
+    ```sh
+    curl --request PATCH \
+        --url https://{region}.api.konghq.com/v2/api-products/{ApiProductId} \
```

Comment:
'ApiProductId' should just be 'id' right? https://docs.konghq.com/konnect/api/api-products/latest/#/API%20Products/update-api-product